readme.md

```diff
@@ -176,10 +176,13 @@ module.exports = () => {
 - communicating between the front and back: can be done using the ipcMain module from electron. See `utils.js` file and example in `navigation` plugin.
 
 ## Build
-
-```sh
-yarn build
-```
+1. Clone the repo
+2. Run ```yarn```
+3. Run ```npm i rimraf```
```

Comment:
rimraf is already a dependency so this step should not be needed!

Reply:
> rimraf is already a dependency so this step should not be needed!

Removed this line!